actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -756,6 +761,13 @@ private void createUser(Request actorMessage) {
       }
       userMap.remove(JsonKey.ORG_EXTERNAL_ID);
       userMap.put(JsonKey.ORGANISATION_ID, orgId);
+
+      // Fetch locationids of the suborg and update the location of sso user
+      OrgService orgService = OrgServiceImpl.getInstance();
```

Comment:
We have to add locationIds to user from org location only if its state user. For Custodian user we dont have to update.